www/docs/main/concepts.mdx

```diff
@@ -7,24 +7,26 @@ slug: /concepts
 
 import { ConceptsChart } from '../../src/components/ConceptsChart';
 
-## What is tRPC? What mindset should I adopt?
+## What is RPC? What mindset should I adopt?
 
 ### It's just functions
 
-tRPC is short for "Typed Remote Procedure Call". It is a way of calling functions on one computer (the server) from another computer (the client). With traditional HTTP/REST APIs, you call a URL and get a response. With RPC, you call a function and get a response.
+RPC is short for "Remote Procedure Call". It is a way of calling functions on one computer (the server) from another computer (the client). With traditional HTTP/REST APIs, you call a URL and get a response. With RPC, you call a function and get a response.
 
 ```ts
 // HTTP/REST
 const res = await fetch('/api/users/1');
 const user = await res.json();
 
-// tRPC
-const user = await api.query('getById',{ id: 1 });
+// RPC
+const user = await api.users.getById({ id: 1 });
 ```
 
+tRPC (Typed Remote Procedure Call) is one implementation of RPC, designed for TypeScript monorepos. It has its own flavor, but is RPC at its heart.
+
 ### Don't think about HTTP/REST implementation details
 
-If you inspect the network traffic of a tRPC app, you'll see that it's just fairly standard HTTP requests and responses. But you don't need to think about the implementation details while writing your application code. You call functions, and tRPC takes care of everything else.
+If you inspect the network traffic of a tRPC app, you'll see that it's fairly standard HTTP requests and responses, but you don't need to think about the implementation details while writing your application code. You call functions, and tRPC takes care of everything else. You should ignore details like HTTP Verbs, since they carry meaning in REST APIs, but in RPC form part of your function names instead, for instance: `getUser(id)` instead of `GET /users/:id`.
 
 ## Vocabulary
```

Comment:
This is all new, 50% covering an omission that likely caused the confusion leading to the first MR. 50% a new nugget based on help questions over the last 6 months